### ISSUE TITLE
The About panel wrongly claims that Graphviz 2.50 is incompatible...

### ISSUE THREAD

**Issue by martinfabian** [Chalmers-Control-Automation-Mechatronics/Supremica#141]
The About panel claims that the Graphviz version I use, 2.50, is "incompatible", but it works just fine here. For a long time we could not use Graphviz beyond version 2.28, but Jonas fixed that in 2019 (see commit fb867bf6b).

Graphviz is now in version 3 (https://graphviz.org/download/), which has not been tested, but 2.50 is the last with major version number 2, and this works (at least on windows).

Bottom line is we should not claim that version 2.x is "incompatible". And we do not know about version 3, so we should probably not say anything about that either.
